libs/components/src/lib/toggletip/toggletip.template.ts

```diff
@@ -0,0 +1,38 @@
+import { html, when } from '@microsoft/fast-element';
+import type { ViewTemplate } from '@microsoft/fast-element';
+import type { ElementDefinitionContext, FoundationElementDefinition } from '@microsoft/fast-foundation';
+import { classNames } from '@microsoft/fast-web-utilities';
+import { Popup } from '../popup/popup';
+import type { Toggletip } from './toggletip';
+
+const getClasses = (_: Toggletip) => classNames('control');
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#Toggletip} component.
+ *
+ * @param context
+ * @public
+ */
+export const ToggletipTemplate:
+(context: ElementDefinitionContext, definition: FoundationElementDefinition)
+=> ViewTemplate<Toggletip> = (context: ElementDefinitionContext) => {
+	const popup = context.tagFor(Popup);
+
+	return html`
+		<${popup}
+			class="${getClasses}"
+			arrow
+			:anchor="${x => x.anchor}"
+			:open="${x => x.open}"
+			?alternate="${x => !x.alternate}"
+			placement="${x => x.placement}"
+			exportparts="vvd-theme-alternate"
+		>
+			<span class="content-wrapper">
```

Comment:
```suggestion
			<div class="content-wrapper">
```
span is an inline-element it shouldn't contain block elements

Reply:
thanks for the comments Rach.

1. sorry I don't think I understand this one though. the toggletip uses the default slot for authors to put whatever structure they want in.
2. I can sure use `header` and `footer`, good point.
3. this was discussed in the [old PR](https://github.com/Vonage/vivid-3/pull/843#issuecomment-1322199844). tl;dr: toggletips don't have a close button, nor does [our design](https://www.figma.com/file/JJNgZvt1qf3ydYmOwbE3Jg/Vivid-UI-Kit---3.0-WIP?type=design&node-id=20630-129571&t=H3XXGTfacvmHIejX-0).